src/riak_object.erl

```diff
@@ -80,6 +80,7 @@
 -define(MAGIC, 53).      %% Magic number, as opposed to 131 for Erlang term-to-binary magic
                          %% Shanley's(11) + Joe's(42)
 -define(EMPTY_VTAG_BIN, <<"e">>).
+-define(MSGPACK_MAGIC, 2). %% Magic number for msgpack encoding
```

Comment:
Minor quibble: Why are we using 'magic' in a macro? It's improved by saying it's part of msgpack, but wouldn't 'msgpack_format_flag' be more informative? (I do see 'magic' on line 80, but that's a legacy, and so I see it as outside scope of this pr).